java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -174,11 +174,13 @@ class ArcHostContextParticle(
     val onReadyJobs = mapOf(
       "particles" to Job(),
       "arcHostContext" to Job(),
-      "handleConnections" to Job()
+      "handleConnections" to Job(),
```

Comment:
Is there a test we can add that would have caught this?

Reply:
Probably not. The more long-term solution would be to finish up the work I was trying to get through before moving back to the TS code: make non-particle handles automatically synchronised so these types of wait checks (including all the `awaitReady` hooks) are no longer needed.